x/builder/ante/ante.go

```diff
@@ -93,10 +93,28 @@ func (ad BuilderDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool,
 			}
 		}
 
-		if err := ad.builderKeeper.ValidateBidInfo(ctx, topBid, bidInfo); err != nil {
+		if err := bd.builderKeeper.ValidateBidInfo(ctx, topBid, bidInfo); err != nil {
 			return ctx, errors.Wrap(err, "failed to validate auction bid")
 		}
 	}
 
 	return next(ctx, tx, simulate)
 }
+
+// ValidateTimeout validates that the timeout is greater than or equal to the expected block height
+// the bid transaction will be executed in.
```

Comment:
Let's add a TODO here with a reference to the issue that tracks that this should be replaced with a Pre-Commit hook that purges stale txs from the mempool.